sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/RestrictDeserializedTypes.LosFormatter.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Web.UI;
+
+namespace SonarAnalyzer.UnitTest.TestCases
+{
+    internal class RestrictDeserializedTypes
+    {
+        public void DefaultConstructor()
+        {
+            new LosFormatter(); // Noncompliant - MAC filtering should be enabled
```

Comment:
Note: the message here will still be `Restrict types of objects allowed to be deserialized.`. Is this wanted or should we change it to `MAC filtering should be enabled` ?

Reply:
Indeed. I've checked also with @eric-therond-sonarsource and we will display a different message for `LosFormatter`: "Serialized data signature (MAC) should be verified."

Reply:
If this is fine with you I would do this change in the following PR (the one with secondary locations) to avoid git re-base conflicts.

Reply:
Ok